section4/README.md

```diff
@@ -50,27 +50,27 @@ Create a new Gist. In it, write a letter introducing yourself to your future ins
 
 - Introduce yourself and share your “why” for taking this step to change your career.
 - What are your strengths? What commitments can you make as far as what/how you will contribute to the cohort and classroom?
-- What do your instructors need to know about you (consider including your workoing styles, your values, how you like to receive feedback, anything else)?
+- What do your instructors need to know about you (consider including your working styles, your values, how you like to receive feedback, anything else)?
 - What did you learn about yourself, as a person and a student, during Mod 0?
 
 Make sure to include your full name and pronouns.
 
-- `Add Link to Gist that contains your letter - here`
+- https://gist.github.com/awirth224/2a9c1254843b93252833009991b65e22
 
 ## Mentorship Prep
 
 Mentorship is an integral part of the Turing experience and will help jumpstart your technical career. In order to get your mentor relationship started on the right foot, please complete the following deliverables:
 
-- [ ] [Create a profile on Turing Community Connect by following these directions](https://docs.google.com/document/d/1vpyKGu92l1HGkJzULNcyyE72946f4QO1DhQgIz3v1E0/edit?usp=sharing)
-- [ ] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
-  - [ ] Add link to your gist here: 
+- [x] [Create a profile on Turing Community Connect by following these directions](https://docs.google.com/document/d/1vpyKGu92l1HGkJzULNcyyE72946f4QO1DhQgIz3v1E0/edit?usp=sharing)
+- [x] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
+  - [x] Add link to your gist here: https://gist.github.com/awirth224/2f6f2a4f73a75da73c2b189b3d635b86
```

Comment:
This is a good start! I like your mentor intro at the end. As you get started in the next few weeks, I'd encourage you to spend some time on the questions you haven't answered yet. I think you will get a clearer idea of where you need technical help with your mentor's help, too!

ps networking is _hard_ but definitely valuable. In my experience, it's more fun if I focus more on finding inspiration from other folks' journeys. And like you said, focus way less on using people to get ahead 🙂